superset-frontend/src/components/TableSelector/index.tsx

```diff
@@ -382,9 +386,8 @@ const TableSelector: FunctionComponent<TableSelectorProps> = ({
         <FormLabel>
           {t('See table schema')}{' '}
           {schema && (
-            <small>
-              {tableOptions.length} in
-              <i>{schema}</i>
+            <small css={smallColor}>
+              {tableOptions.length} {t(' ')} in {schema}
```

Comment:
you don't need to wrap the space in the translate function `t(' ')`

Reply:
actually since they're on the same line now, you should be able to use a normal space without the implicit string {' '}.
`{tableOptions.length} in {schema}`